tools/psa/release.py

```diff
@@ -112,7 +113,22 @@ def _get_default_image_build_command(target, toolchain, profile):
     else:
         cmd += ['--artifact-name', 'psa_release_1.0']
 
-    return cmd
+    return cmd + args
+
+
+def verbose_check_call(cmd, check_call=True):
+    """
+
```

Comment:
Was there suppose to be a comment before this line?

Reply:
Added missing Docstring